### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by RenanUmV**

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:help

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:startup

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:startup

**Comment by RenanUmV** [comment]
ci:test:lima

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portlet/ServerStatistics.java [hunk 1]
```diff
@@ -35,78 +30,30 @@
  */
 @Component(
 	enabled = false, property = "namespace=com.liferay.monitoring.Portlet",
-	service = {DataSampleProcessor.class, ServerStatistics.class}
+	service = DataSampleProcessor.class
 )
 public class ServerStatistics
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portlet/ServerStatistics.java:35]: Hey @RenanUmV, rename this class to reflect your changes. Thx

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatistics.java [hunk 2]
```diff
@@ -27,72 +25,11 @@
  */
 @Component(
 	enabled = false, property = "namespace=com.liferay.monitoring.Service",
-	service = {DataSampleProcessor.class, ServerStatistics.class}
+	service = ServerStatistics.class
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatistics.java:28]: Hey @RenanUmV, you removed the wrong service. Please fix that.

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatistics.java [hunk 3]
```diff
@@ -27,72 +25,11 @@
...
 )
 public class ServerStatistics
 	implements DataSampleProcessor<ServiceRequestDataSample> {
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatistics.java:31]: Rename this class as well

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/util/ServerStaticsUtil.java [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+/**
+ * @author Renan Vasconcelos
+ */
+public class ServerStaticsUtil {
+
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/util/ServerStaticsUtil.java:31]: Hey @RenanUmV, in this case, taking into account the `companyLocalService` reference you should change this to a helper class instead. Thx

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/ServerStatistics.java [hunk 5]
```diff
@@ -35,65 +30,27 @@
  */
 @Component(
 	enabled = false, property = "namespace=com.liferay.monitoring.Portal",
-	service = {DataSampleProcessor.class, ServerStatistics.class}
+	service = ServerStatistics.class
 )
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/ServerStatistics.java:34]: Hey @RenanUmV, please rename and use the proper service. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @RenanUmV, Could you please try to unify the ServerStaticsUtil classes into only one util/helper class?

Thx

**Pull request closed without merging**